src/apps/VirtualApp/VirtualChat.js

```diff
@@ -48,7 +48,7 @@ class VirtualChat extends Component {
       },
       error: (reason) => {
         console.error("  -- Error join room", reason);
-        reportToSentry(reason, {source: "Textroom"});
+        reportToSentry(reason, {source: "Textroom", reason});
```

Comment:
Extra data sent to sentry should be consistent. So does the code itself.

- Argument to error callback should be named `err`. So does the key in sentry extra data
- janus message handlers should be named `msg`